lab_4_fill_words_by_ngrams/main.py

```diff
@@ -28,6 +32,20 @@ def _tokenize(self, text: str) -> tuple[str, ...]:  # type: ignore
         Raises:
             ValueError: In case of inappropriate type input argument or if input argument is empty.
         """
+        if not (isinstance(text, str) and text):
+            raise ValueError ("WordProcessor._tokenize: inappropriate input")
```

Comment:
remove the prefix in the message. This information should be in traceback

Reply:
![OkAlrightThenGIF](https://github.com/fipl-hse/2023-2-level-labs/assets/144431295/430bb94c-6a12-4e95-beef-55938cc4878b)